svd-rs/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## [v0.14.5] - 2023-11-22
+
 - `default_value` for `EnumeratedValues`
 - move `EnumeratedValue` check to `strict` mode and fix it
```

Comment:
hmm reading this, is this actually true?

Reply:
what you mean?

Reply:
it's too broad, it should say something like `moved isDefault and value check on enumerated values to strict mode`